RELEASES.md

```diff
@@ -0,0 +1,59 @@
+
+### Major and Minor Releases
+Major and minor releases of Velero will be branched from `main` when the release reaches to `RC(release candidate)` state. The branch format should follow `release-<major>.<minor>.0`. For example, once the release `v1.0.0` reaches to RC, a branch will be created with the format `release-1.0.0`. When the release reaches to `GA(General Available)` state, The tag with format `v<major>.<minor>.<patch>` and should be made with command `git tag -s v<major>.<minor>.<patch>`. The release cadence is around 3 months, might be adjusted based on open source event, but will communicate it clearly.
+
+### Patch releases
+Patch releases are based on the major/minor release branch, the release cadence for patch release of recent minor release is one month to solve critical community and security issues. The cadence for patch release of recent minus two minor releases are on-demand driven based on the severity of the issue to be fixed.
+
+### Pre-releases
+`Pre-releases:mainly the different RC builds` will be compiled from their corresponding branches. Please note they are done to assist in the stabilization process, no guarantees are provided.
+
+### Minor Release Support Matrix
+| Version | Supported          |
+| ------- | ------------------ |
+| Velero v1.10.x   | :white_check_mark: |
+| Velero v1.9.x   | :white_check_mark: |
+| Velero v1.8.x   | :white_check_mark: |
+
+### Upgrade path and support policy
+The upgrade path for Velero is (1) 2.2.x patch releases are always compatible with its major and minor version. For example, previous released 2.2.x can be upgraded to most recent 2.2.3 release. (2) Velero only supports two previous minor releases to upgrade to current minor release. For example, 2.3.0 will only support 2.1.0 and 2.2.0 to upgrade from, 2.0.0 to 2.3.0 is not supported. One should upgrade to 2.2.0 first, then to 2.3.0.
+The Velero project maintains release branches for the three most recent minor releases, each minor release will be maintained for approximately 9 months.
+
+### Next Release
+The activity for next release will be tracked in the [Velero Wiki roadmap](https://github.com/vmware-tanzu/velero/wiki). If your issue is not present in the corresponding release, please reach out to the maintainers to add the issue to the project board.
+
+### New Features
+All features(Issues and PRs) that are aimed to be completed into the upcoming release must be labeled as `X.YY-cadidate`, where X.YY is the upcoming version.
+All features that were not completed it into the upcoming release needs to be revisited and re-labeled for the next release, if they are still considered valid for the project! The decision will be made by the major stakeholders(maintainers).
+
```

Comment:
Shall we also explain the practice we are taking in the development of the next release?

i.e.
We regularly triage the issues with `X.YY-candidate` and for the ones we plan to work on in that release we put them into  at the milestone of this release, for example:
https://github.com/vmware-tanzu/velero/milestone/47
At the milestone `Feature Freeze of X.YY` all the labels `X.YY-candidate` will be removed and the issues will be either moved into the milestone or push out of the `X.YY` release.

Reply:
current should be 1.10 not 1.11, 1.11 is a candidate/upcoming/next right ?

Reply:
Right. 1.11 doesn't exist yet. It's not supported yet. Current/latest release is 1.10.

Reply:
> current should be 1.10 not 1.11, 1.11 is a candidate/upcoming/next right?

Yes v1.11 is the upcoming release.

Reply:
@OrlinVasilev I propose below:
1.	All features (Issues and PRs) that are aimed to be completed into the upcoming release must be labelled as ‘X.YY-candidate’, where ‘X.YY’ is the upcoming version.
2.	Velero maintainers regularly triage the issues labelled with ‘X.YY-candidate’. The ones which will be planned for ‘X.YY’ release, maintainers will add milestone X.YY and remove ‘X.YY-candidate’ label from them.
3.	After the feature freeze date for the X.YY release, X.YY-candidate label will be removed from all the issues. Maintainers may re-label these issues for next Velero release candidate.

Also, Velero project is driven by rich community, lets not put commitment to any fixed number of release per year. I am sure community will drive the regular releases. Shall we just say, "Velero team is aiming for 3  releases per year with few patch releases in between."

@reasonerjt @sseago WDYT?

Reply:
Sounds good to me.

Reply:
all adopted